src/p2p/behaviour.rs

```diff
@@ -464,6 +464,10 @@ impl<Types: IpfsTypes> Behaviour<Types> {
         &mut self.bitswap
     }
 
+    pub fn ipfs(&self) -> &Ipfs<Types> {
+        &self.ipfs
+    }
+
```

Comment:
This doesn't seem really too good.

Reply:
I was trying to follow the same logic as with `swarm()` and `bitswap()` here, but this will not be needed as soon as [this alternative](https://github.com/rs-ipfs/rust-ipfs/pull/110#discussion_r470702916) is implemented.